<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTStoreReport</class>
 <widget class="QWidget" name="FTStoreReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="WDate2" name="wd" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Partner</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="lePartner">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Store</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leStore">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Goods</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leGoods">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>290</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>WDate2</class>
   <extends>QWidget</extends>
   <header location="global">wdate2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
